kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDeviceAddedHandler.java

```diff
@@ -33,7 +33,8 @@ public NMDeviceAddedHandler(NMDbusConnector connector) {
     public void handle(NetworkManager.DeviceAdded s) {
         try {
             logger.info("New network device connected at {}", s.getDevicePath());
-            this.nm.apply();
+            String deviceId = this.nm.getDeviceIdByDBusPath(s.getPath());
```

Comment:
```suggestion
            String deviceId = this.nm.getDeviceIdByDBusPath(s.getDevicePath());
```

Reply:
Is`s.getDevicePath().getPath()` better than the `toString()` method?